sound/components/virt.c

```diff
@@ -0,0 +1,248 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <microkit.h>
+#include <sddf/sound/queue.h>
+
+#define DRIVER_CH 3
+#define CLIENT_CH_BEGIN 1
+#define CLIENT_COUNT 2
```

Comment:
```suggestion
#define NUM_CLIENTS 2
```

just to be consistent with the other device classes.